doc/source/reference/commands/logical_range_filter.rst

```diff
@@ -279,7 +279,15 @@ TODO: Add examples
 ``order``
 """"""""""
 
-TODO
+Specifies order of search result.
+Specify ``ascending`` or ``descending`` as the value of this parameter.
+
+If we set ``ascending`` in this parameter, search results are ordered ascending base on ``shard_key``.
```

Comment:
It seems that `search results are sorted by ascending order based on shared_key` or something is better.